docs/src/user/tasks.rst

```diff
@@ -102,74 +102,209 @@ Prediction Format
 
 A array of integers representing the list of predicted tag for each token, in sequence, for the sentence.
 
-TABULAR_CLASSIFICATION
+
+
+QUESTION_ANSWERING
 --------------------------------------------------------------------
 
-Dataset Type
+COVID19 Task Dataset Format
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-:ref:`dataset-type:TABULAR`
+:ref:`dataset-type:QUESTION_ANSWERING`
```

Comment:
is `dataset-type` used in the code?
if yes, you may need to change it to QUESTION_ANSWERING_COVID19.

Reply:
'dataset-type' is not appearing incode. it is for referring to page  https://nginyc.github.io/rafiki/docs/latest/src/user/datasets.html#dataset-type-image-files, which shall be updated also